packages/edit-navigation/src/components/menu-editor/style.scss

```diff
@@ -62,24 +62,30 @@
 	font-weight: bold;
 }
 
-.edit-navigation-menu-editor__block-editor-panel {
+.edit-navigation-menu-editor__block-editor-area {
 	@include break-medium {
 		// IE11 requires the column to be explicitly declared.
 		// Only shift this into the second column on desktop.
 		grid-column: 2;
 	}
 
-	// Make panels collapsible in IE. The IE analogue of align-items: self-start;.
-	-ms-grid-row-align: start;
+	.components-card__footer,
+	.components-card__header {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+	}
```

Comment:
should these be bundled in the Card component or not? Genuinely asking, I don't have the answer but if I understand properly when we find ourselves doing this kind of override (which we do often), maybe we're missing something in the abstraction.

Reply:
Thanks for the heads up @youknowriad !

We recently merged in an update that added a new layout-based component, `<Flex />`, that handles this very use-case!

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/flex

I can update the `<Card />` sub-components to use `<Flex />`, which should save us from having to write CSS like this in future use